crates/pathfinder/src/rpc.rs

```diff
@@ -55,6 +56,8 @@ impl<Context: Send + Sync + 'static> RpcModuleWrapper<Context> {
     {
         use tracing::Instrument;
 
+        metrics::register_counter!(format!("{method_name}_calls_total"));
```

Comment:
Checked that naming seems to follow https://prometheus.io/docs/practices/naming/ except perhaps the namespacing, shouldn't it be like `rpc_starknet_syncing_calls_total`? In their naming examples they have `api_` or:

> should have a (single-word) application prefix relevant to the domain the metric belongs to. The prefix is sometimes referred to as `namespace` by client libraries.

As for should it be `api` or `rpc` ... I don't have strong opinion but I'd lean towards `rpc_`.

Reply:
I think let's use @kkovaacs 's (offline) suggestion to use `rpc_method_calls_total` with a label `"method" => "method_name_here"`.